automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala

```diff
@@ -219,9 +222,7 @@ class DataSpec extends FreeSpec with WebBrowserSpec
           val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProject, workspaceName, SimpleMethodConfig.configNamespace, methodConfigName).open
           val submissionTab = methodConfigDetailsPage.launchAnalysis(SimpleMethodConfig.rootEntityType, TestData.SingleParticipant.entityId, "", false)
           submissionTab.waitUntilSubmissionCompletes()
-          if (submissionTab.readWorkflowStatus() != "Succeeded") {
-            submissionTab.readStatusMessage() shouldEqual ""
-          }
```

Comment:
I don't understand what the above conditional is doing. If the workflow status isn't a success the test would fail, right? Why would we be testing for an unexpected state?